airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/LowCreditBalanceHint/LowCreditBalanceHint.tsx

```diff
@@ -0,0 +1,37 @@
+import { faCreditCard, faWarning } from "@fortawesome/free-solid-svg-icons";
+import classNames from "classnames";
+import { FormattedMessage } from "react-intl";
+
+import { InfoBox } from "components/ui/InfoBox";
+
+import { useGetCloudWorkspace } from "packages/cloud/services/workspaces/CloudWorkspacesService";
+import { useCurrentWorkspace } from "services/workspaces/WorkspacesService";
+
+// eslint-disable-next-line css-modules/no-unused-class
+import styles from "./LowCreditBalanceHint.module.scss";
+
+export const LOW_BALANCE_CREDIT_TRESHOLD = 20;
+
+export const LowCreditBalanceHint: React.FC = ({ children }) => {
```

Comment:
```ts
export const LowCreditBalanceHint: React.FC<React.PropsWithChildren<unknown>> = ({ children }) => {
```

Adding the `PropsWithChildren` will ease the migration to React 18 later on. So we should always wrap the actual props in it when accessing `children` (or `unknown` in case we don't have any other props).